app/controllers/steps/postcode_screen_step_controller.rb

```diff
@@ -0,0 +1,9 @@
+module Steps
+  class PostcodeScreenStepController < StepController
+    private
+
+    def decision_tree_class
+      C100App::PostcodeScreenDecisionTree
```

Comment:
I've realised that what we probably want is a "Screener" task, with many steps, as the screener will be composed of multiple steps, not only the postcode step. That way we have a single decision tree handling all the screener logic.

Reply:
That's true, but we do also need a separate postcode step for near the end of the process, when we tell the user where they need to send their printed application.

Reply:
So the way I understand it, is a Task screener, composed of N-steps, one of them being the postcode 'whitelisting', and if the postcode passes our whitelist/rules for the pilot, we have it already stored in DB and by the end of the process (granting the rest of the screening questions were successful), based on this postcode, we show the correct instructions. But there is no separate step for postcode at the end, is just the 'Completion' step, with instructions.

Reply:
my thinking was that in the Completion step, you'd have something like:

What to do now
===========
Print the form, sign it, and post it to:

<the appropriate court address>

(button: Print the form)